GameOfLife/View/GameParameters.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// Contains defined minimal and maximal game parameters
+    /// </summary>
+    public class GameParameters
+    {
+        public static int minInputValue = 5;
```

Comment:
Those variables shouldn't be changing during execution of application so make them constant.